examples/checkout/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
```

Comment:
Please add `<meta name="viewport" content="width=device-width">` here for both examples. Just noticed the cart example currently isn't responsive.